main/src/ca/uwaterloo/flix/api/lsp/PrettyPrinter.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2025 gwydd
```

Comment:
Add your real name (necessary for copyright). But you are welcome to put gwydd in parens after.